models/engine/db_storage.py

```diff
@@ -74,3 +74,14 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieves the object"""
+        name = cls + "." + id
+        if name in self.all():
+            return self.all().get(key)
```

Comment:
Nice clean and precise function! However, was the key variable a typo? I only see the name variable defined.